### ISSUE TITLE
Interpolation Method

### ISSUE THREAD

**Issue by emma994** [Riverscapes/gcd#338]
When uploading a DEM, in the 'Specify GCD DEM Survey' box there used to be a drop down on the 'interpolation method' option. This no longer seems to be there and it is default to 'bilinear interpolation'. I think this is causing my DEM to become mis-shaped as it is not a straight cell wise copy

What went wrong?
You can see from the attachment that the original DEM of the river channel (in black and white) looks nothing like the output after uploading into GCD.

![emmademtest](https://user-images.githubusercontent.com/41896662/51529773-50d6dc80-1e31-11e9-9524-c6b70c4f29ed.JPG)

**Comment by philipbaileynar** [comment]
@emma994 this is almost certainly a map projection issue. Try the following:

1. Start a new GCD project.
1. Find your first DEM survey raster and verify that it is in a **projected** coordinate system. i.e. the linear units should be metres or feet etc **not decimal degrees**.
1. Import this verified raster into your GCD project. If the edges of the raster are not evenly divisible by the cell resolution then GCD will have to perform a bilinear resample, but this should not dramatically alter the shape of your data. 
1. Repeat steps 2 and 3 for each subsequent raster that you want in your GCD project.

In summary, I suspect that some of your rasters are projected while others are in geographic coordinate systems. Are you getting any GCD warnings about mis-matching coordinate systems?

Let me know if this doesn't solve the problem.

**Comment by philipbaileynar** [comment]
Moving @emma994  subtraction problem to new issue #340 and closing this issue because it is resolved.
